services.rst

```diff
@@ -64,48 +64,67 @@ server in the NTP configuration page.
+workgroup. For an AD domain of :code:`samdom.example.com`, for instance, the
+Samba workgroup should be set as :code:`SAMDOM` as illustrated below. Note that
+while the Samba service needs to be configured, it doesn't need to be turned
+ON.
 
-.. image:: ad-config.png
+.. image:: images/services/ad_samba_config.png
    :scale: 70 %
    :align: center
 
-The individual fields of the form are described below.
-
-* **Domain**: Specifies the Windows Active Directory or domain controller to
-  connect to.
-* **Controllers**: domain controller to use.
-* **Security**:  The security model to use, which configures how clients should
-  respond to Samba. The options are
-
-   1. user. A client must first log in with a valid username and password.
-   2. server. In this mode, Samba will attempt to validate the username/password by authenticating it through another SMB server (for example, a Windows NT Server). If the attempt fails, the user mode will take effect instead.
-   3. domain. In this mode, Samba will attempt to validate the username/password by authenticating it through a Windows NT Primary or Backup Domain Controller, similar to how a Windows NT Server would.
-   4. ads. This mode instructs Samba to act as a domain member in an Active Directory Server (ADS) realm.
+Next, as correct time synchronization with the AD domain is necessary for good
+performance, the NTP service should be configured and turned ON. Ideally, both
+the AD domain and Rockstor machines should thus use the same NTP time server.
 
-* **Realm**: When the ads Security Model is selected, this allows you to
-  specify the ADS Realm the Samba server should act as a domain member of.
-* **Template shell**: The login shell for the user
-* **Allow offline login**
+Now that the Samba workgroup has been defined, and the NTP service configured
+and running, the AD service can be configured. To do so, click on the
+**wrench** icon and fill the form with the values corresponding to your AD
+domain.
 
-To start or stop the service, click the corresponding ON or OFF button.
-
-For example, for an environment with the Winbind domain as *rockstorad*, and the AD controller as *WIN-H323VCUT6GT.ROCKSTORAD.LOCAL*, these are the values used to configure AD.
+.. image:: images/services/ad_config.png
+   :scale: 70 %
+   :align: center
 
-    * Winbind Domain - rockstorad
-    * Security Model - ads
-    * Winbind ADS Realm - rockstorad.local
-    * Winbind Domain Controllers - WIN-H323VCUT6GT.ROCKSTORAD.LOCAL
-    * Template shell - /bin/bash
-    * Don't check Allow offline login box.
+The individual fields of the form are described below.
 
-Once the AD service is configured, click *Join Domain* and enter the AD Administrator username and password, and click Submit. The appliance will now attempt to join the domain and will display a *Join Ok* message if successful.
+* **Domain/Realm name**: Specifies the Windows Active Directory or domain
+  controller to which to connect.
```

Comment:
```suggestion
* **Domain/Realm name**: Specifies the desired Active Directory or Domain.
```
Shorter with implied connect which will hopefully be obvious enough. May also translate easier.